src/AzureClient/EntryPoint/EntryPoint.cs

```diff
@@ -4,14 +4,26 @@
 #nullable enable
 
 using System.IO;
+using System.Reflection.PortableExecutable;
 using System.Threading;
 using Microsoft.Extensions.Logging;
+using Microsoft.Quantum.QsCompiler;
 using Microsoft.Quantum.Runtime;
 using Microsoft.Quantum.Runtime.Submitters;
 using Microsoft.Quantum.Simulation.Core;
 
 namespace Microsoft.Quantum.IQSharp.AzureClient
 {
+    public record EntryPointSubmitArgs
```

Comment:
nit: the name can be confused with something that is actually part of the job itself rather than the payload of a telemetry event. Maybe it could be named something like `EntryPointSubmitEventData` or `EntryPointSubmitEventPayload`.